releases/v1.1.0.toml

```diff
@@ -0,0 +1,150 @@
+
+### Registry Mirror
+
+You can now setup registry configurations with the config option
+`plugins.cri.registry`.
+
+Currently only the `mirrors` option is supported. With it, you can specify
+registry mirrors and secure/insecure connections.
+([doc](https://github.com/containerd/cri/blob/v1.0/docs/registry.md))
+
+### End-To-End Test
+
+In terms of testing, we've passed:
+* ALL CRI validation tests
+* ALL node e2e tests
+* ALL e2e tests
+
+The containerd test coverage on GCE is equivalent with Docker now.
+
+All the test results are public: https://k8s-testgrid.appspot.com/sig-node-containerd.
+
+### Performance
+
+We significantly improved pod start latency and cpu/memory usage of `cri` plugin
+this release.
+
+The continuous benchmark result is published on http://node-perf-dash.k8s.io/.
+Job `ci-kubernetes-node-kubelet-benchmark` is for Docker 17.03, and
+`ci-cri-containerd-node-e2e-benchmark` is for containerd with `cri` plugin.
+
+All metrics of containerd are either better or comparable with Docker 17.03.
+
+## Try It Out
+
+If you would like to try containerd, please download the binaries included on
+this release. If you are using Docker, this version of containerd will be used
+in the next major release of Docker.
+
+To set up containerd with Kubernetes...
+* For a production quality cluster on GCE brought up with `kube-up.sh`, see [here](https://github.com/kubernetes-incubator/cri-containerd/blob/v1.0/docs/kube-up.md).
+* For a multi-node cluster installer and bring up steps using ansible and kubeadm, see [here](https://github.com/kubernetes-incubator/cri-containerd/blob/v1.0/contrib/ansible/README.md).
+* For creating a cluster from scratch on Google Cloud, see [Kubernetes the Hard Way](https://github.com/kelseyhightower/kubernetes-the-hard-way).
+* For a custom installation from release tarball, see [here](https://github.com/kubernetes-incubator/cri-containerd/blob/v1.0/docs/installation.md).
+* To install using LinuxKit on a local VM, see [here](https://github.com/linuxkit/linuxkit/tree/master/projects/kubernetes).
+
+## Support
+
+The [_support horizon_](https://github.com/containerd/containerd/blob/master/RELEASES.md#support-horizon)
+for containerd has been updated to include the 1.1 release. With the addition of
+the CRI plugin, we are expanding the support horizon for 1.1 to include the
+entire lifespan of Kubernetes 1.10.  The containerd 1.1 release train will be
+considered an active branch with new patches until April 23, 2019 at the
+earliest, when 1.2 is released, or until Kubernetes 1.10 reaches end of life.
+The 1.1 containerd API is completely compatible with 1.0, any client using 1.0
+can safely upgrade to 1.1 without any incompatibilies. The CRI interface
+included with the CRI plugin is only supported for Kubernetes 1.10. The CRI
+interface is still considered alpha and will only be supported for Kubernetes
+1.10. The CRI plugin is version 1.0 in containerd 1.1 will also only be
```

Comment:
@Random-Liu I added some wording here about 1.0 and `containerd/cri` only being supported in containerd 1.1. SGTY?

Reply:
I feel like we should try to make user forget about `containerd/cri` version, all the new features of `containerd/cri` are `containerd` new features, and should be mentioned in and bound to containerd release. The version in `containerd/cri` is only used to manage the project.

Maybe we should not include `cri` version here to make it less confusing?